### DISCUSSION TITLE
Support for maximum discount in percentage vouchers

### DISCUSSION THREAD

**Discussion by akhial** [saleor/saleor#14431] [Ideas]
### What I'm trying to achieve
Cap the discount for a given percentage voucher. For example a 50% voucher with 10$ maximum would discount:
- 4$ from a 8$ checkout
- 10$ from a 100$ checkout

### Describe a proposed solution
Add maximum discount field to `Voucher` and related mutations and queries
Add input for maximum discount in dashboard

### Mockup
![max](https://user-images.githubusercontent.com/20596622/191217871-5fd60335-609e-44d1-ad73-9e630d71c8d4.jpg)

**Comment by peelar** [comment]
Hello @akhial 👋 

Thank you for your feature request. We are currently evaluating whether this is something we are interested in bringing to Saleor. We will let you know about the result.

Nevertheless, we are grateful that you are participating in the development of Saleor 🙇.
